### ISSUE TITLE
System.Security.Cryptography.Xml v4.4.2 NuGet package runtime load failure on .NET Core SDK 2.1.201

### ISSUE THREAD

**Issue by Heronyme** [dotnet/corefx#29953]
When using the latest stable .NET Core SDK (v2.1.201) there is an assembly load failure when referencing System.Security.Cryptography.Xml v4.4.2 (latest stable) obtained via NuGet package. 

No errors with v2.1.200 SDK, but then was kinda forced to upgrade to v.2.1.201 by a VS2017 update and now the official download link also contains this version.

Is this behaviour expected? It appears that the pre-release version of the package (4.5.0-rc1) works with SDK v2.1.201, but we try to avoid pre-release versions. 

Steps to reproduce:

1. Create new .NET Core 2.0 Console Application project
2. Add System.Security.Cryptography.Xml v4.4.2 package
3. Reference it in the Main method, e.g. add "var reference = new System.Security.Cryptography.Xml.Reference();"
4. Compile
5. Run

Expected result:

App runs without errors.

Actual result: 

```
Unhandled Exception: System.IO.FileLoadException: Could not load file or assembly 'System.Security.Cryptography.Xml, Version=4.0.0.1, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
   at Cryptography.Xml.Test.Program.Main(String[] args)
```

**Comment by karelz** [comment]
@weshaggard @joperezr is it a known problem? Looks like some packaging hiccup ...

**Comment by Heronyme** [comment]
It seems a new stable System.Security.Cryptography.Xml package was released today (v4.5.0). This resolves the issue for me. Tested to work with both SDK v2.1.200 and v2.1.201. Thank you.
